substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1161,6 +1162,91 @@ impl<T: Config> Pallet<T> {
 	) -> Exposure<T::AccountId, BalanceOf<T>> {
 		EraInfo::<T>::get_full_exposure(era, account)
 	}
+
+	/// Whether `who` is a virtual staker whose funds are managed by another pallet.
+	pub(crate) fn is_virtual_staker(who: &T::AccountId) -> bool {
```

Comment:
This fn seems unrelated to the logic to be implemented?

Reply:
did not touch that actually